shop-be/import-service/package.json

```diff
@@ -4,16 +4,20 @@
   "description": "Serverless aws-nodejs-typescript template",
   "main": "serverless.ts",
   "scripts": {
+    "start": "serverless offline start",
+    "invoke:local:importFileParser": "npx sls invoke local -f importFileParser --path src/functions/importFileParser/mock.json",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "engines": {
     "node": ">=14.15.0"
   },
   "dependencies": {
+    "@aws-sdk/client-sqs": "^3.49.0",
     "@middy/core": "^2.5.3",
     "@middy/http-json-body-parser": "^2.5.3",
     "csv-parser": "^3.0.0",
-    "serverless-finch": "^2.8.0"
+    "pg": "^8.7.1",
+    "serverless-s3-local": "*"
```

Comment:
works with aws-sdk@2.N.N

Reply:
It seems that there is an issue with aws-sdk v3, but I see that there was one proposal with fix - https://github.com/ar90n/serverless-s3-local/issues/415